### PR TITLE
FAQ: add "where to learn elisp" entry

### PR DESCRIPTION
**Pull request by deb0ch**
As this question is frequently asked in the gitter chat, I thought that it might be nice to have an entry for it in the FAQ.

For the very quick tutorial I linked to https://learnxinyminutes.com/docs/elisp/, whereas the original post for that article was in http://emacs-doctor.com/learn-emacs-lisp-in-15-minutes.html. Not sure which one should be in the FAQ, but I like LearnXInYMinutes because it opens to many more languages :smiley: 

Although emacs-doctor also has good arguments in its favor.

I'm really undecided, so just tell me which one you would prefer to be in the FAQ.

### CODE REVIEW

File: doc/FAQ.org [hunk 1]
```diff
@@ -262,6 +263,13 @@ You can also disable this feature entirely by adding =exec-path-from-shell= to
 the list =dotspacemacs-excluded-packages= if you prefer setting =exec-path=
 yourself.
 
+** I want to learn elisp, where do I start ?
+Very quick start: [[http://https://learnxinyminutes.com/docs/elisp/][learn X in Y minutes (where X is elisp)]]
```
Comment (jonaustin) [inline doc/FAQ.org]: extra http:// :)

Comment (deb0ch) [inline doc/FAQ.org]: Nicely spotted, thanks !

**Comment by deb0ch** [comment]
Ping: what is the status on this ?

**Comment by syl20bnr** [comment]
pooooonnnnnnnnnnnggg (very late and low pong :-))
Nice addition 👍 
Thank you !
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by deb0ch** [comment]
Nice ! :thumbsup::sparkles:

```
PING syl20bnr@spacemacs (42.42.42.42) 56(84) bytes of data.
64 bytes from 42.42.42.42: icmp_seq=1 ttl=58 time=5184000000 ms
```

**Pull request closed without merging**